lua/pac3/editor/client/tools.lua

```diff
@@ -276,84 +276,76 @@ do
 
 end
 
-pace.AddTool(L"convert to expression2 holo", function(part)
-	local holo_str =
+pace.AddTool(L"Convert group of models to Expression 2 holograms", function(part)
+	local holo_str = 
 	[[
-
-	HOLO_NAME = IDX
-	holoCreate(HOLO_NAME)
+	
+		holoCreate(I) #HOLO_NAME
 		PARENT
-		holoColor(HOLO_NAME, COLOR)
-		holoAlpha(HOLO_NAME, ALPHA)
-		holoScale(HOLO_NAME, SCALE)
-		holoPos(HOLO_NAME, entity():toWorld(POSITION))
-		holoAng(HOLO_NAME, entity():toWorld(ANGLES))
-		#holoAnim(HOLO_NAME, ANIMATION_NAME, ANIMATION_FRAME, ANIMATION_RATE)
-		#holoDisableShading(HOLO_NAME, FULLBRIGHT)
-		holoMaterial(HOLO_NAME, MATERIAL)
-		holoModel(HOLO_NAME, MODEL)
-		holoSkin(HOLO_NAME, SKIN)
+		holoColor(I, COLOR)
```

Comment:
Cleaned up how the holograms are laid out, now auto indexes and doesn't include unimplemented stuff.